alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/MapDisplay.java

```diff
@@ -60,6 +62,25 @@ public MapDisplay() {
         add(mapView);
     }
 
+    private static TileCache createTileCache() {
+        final TileCache firstLevelTileCache = new InMemoryTileCache(IN_MEMORY_TILES);
+        final String tmpdir = System.getProperty("java.io.tmpdir");
+        final File cacheDirectory = new File(tmpdir, "mapsforge" + IDGEN.getAndIncrement());
+        final TileCache secondLevelTileCache = new FileSystemTileCache(ON_DISK_TILES, cacheDirectory, GRAPHIC_FACTORY);
+        return new TwoLevelTileCache(firstLevelTileCache, secondLevelTileCache);
+    }
+
+    private static TileDownloadLayer createTileDownloadLayer(final TileCache tileCache, final Model model) {
+        final TileSource tileSource = OpenStreetMapMapnik.INSTANCE;
+        final TileDownloadLayer tdl = new TileDownloadLayer(
+                tileCache,
+                model.mapViewPosition,
+                tileSource,
+                GRAPHIC_FACTORY);
+        tdl.setDisplayModel(model.displayModel);
+        return tdl;
+    }
+
```

Comment:
Why did you move these static methods before instance methods?

Reply:
I checked the [blame](https://github.com/Vuksaa/Alchemist/blame/develop/alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/MapDisplay.java#L65) and [past commits](https://github.com/Vuksaa/Alchemist/commit/2b40315ea59127eb2361026bf967460d1d4186ff#diff-d39a906518334a6b6b0725a0d3d73750), these methods were moved because codacy reported some problems.

Reply:
Then codacy and I disagree on how members should get sorted.
Anyway, let's mark as solved.